docs/src/prototypes/chatPane/services/dataMock.ts

```diff
@@ -69,7 +69,7 @@ class ChatMock {
       }
 
       return message
-    }).sort((a, b) => a.date - b.date)
+    }).sort((a, b) => a.date.getMilliseconds() - b.date.getMilliseconds())
```

Comment:
this might introduce unexpected sort result, as you are now comparing just ms part of the dates collection (so that 17:00:0.456 > 17:01:0.123, as 456 > 123). Is it intentional?

Reply:
meant to compare the dates, used `getTime` instead